cmd/truss/main.go

```diff
@@ -29,8 +30,32 @@ var (
 	verboseFlag    = flag.Bool("v", false, "Verbose output with stack traces for errors.")
 )
 
+const (
+	noVersion   string = "<no-version>"
+	noBuildDate string = "<no-build-date>"
+)
+
+var (
+	Version   string = noVersion
+	BuildDate string = noBuildDate
+)
+
 func init() {
+	buildinfo := new(bytes.Buffer)
+	if Version != "" && Version != noVersion {
+		buildinfo.WriteString("version: ")
+		buildinfo.WriteString(Version)
+		buildinfo.WriteByte(' ')
+	}
+	if BuildDate != "" && BuildDate != noBuildDate {
+		buildinfo.WriteString("built: ")
+		buildinfo.WriteString(strings.Replace(BuildDate, "_", " ", -1))
+	}
+
 	flag.Usage = func() {
+		if buildinfo.Len() > 0 {
+			fmt.Fprintf(os.Stderr, "Truss (%s)\n", strings.TrimSpace(buildinfo.String()))
```

Comment:
I feel it is odd to print out capital `Truss` here when the error message for `flag.Args() == 0` will print out the actual name of the binary, probably `truss`. 

What about using the `filepath.Base(os.Args[0])` here as well?